Lib/test/test_capi/test_float.py

```diff
@@ -178,6 +179,38 @@ def test_pack_unpack_roundtrip(self):
                         else:
                             self.assertEqual(value2, value)
 
+    @unittest.skipUnless(HAVE_IEEE_754, "requires IEEE 754")
+    def test_pack_unpack_roundtrip_for_nans(self):
+        pack = _testcapi.float_pack
+        unpack = _testcapi.float_unpack
+        for _ in range(1000):
```

Comment:
100 tests should be enough to validate the implementation, no?

```suggestion
        for _ in range(100):
```

1000 tests might be a little bit too slow, I don't think that it's worth it.

Reply:
Up to you, the test looks instantaneous on my system.  0.3sec vs 0.03.  Where the threshold?

Reply:
I don't think speed is really an issue here, but having the potential for 6000 failed test reports seems like major overkill.  I think 10 would actually be plenty for this loop.